ui/src/app/pages/workspace/create-billing-account-modal.tsx

```diff
@@ -302,8 +305,8 @@ export const CreateBillingAccountModal = ({onClose}: Props) => {
                 </div>
               </FlexColumn>
             </FlexRow>
-            <FlexRow style={{marginTop: '20px'}}>
-              <CheckBox style={{margin: '15px', height: '1.063rem', width: '1.063rem'}}
+            <FlexRow style={{marginTop: '10px'}}>
+              <CheckBox style={{margin: '15px',  width: '17px', height: '17px', flexShrink: 0, maxWidth: '17px'}}
```

Comment:
original comment was to pull these out into a shared style

why maxWidth? flexGrow: 0 is probably the symmetric option if it's necessary

Reply:
ah, I thought your comment want me to use both maxWidth and flexShrink.